### PR TITLE
feat(nx-plugin): use distinct file suffix patterns for playwright component tests

### PR DESCRIPTION
**Pull request by Hotell**
- introduces new suffix pattern
- updates all existing repo files and configs
- tweaks lint configs

Closes https://github.com/microsoft/fluentui-contrib/issues/118

### CODE REVIEW

**Comment by Hotell** [comment]
Jest tests started to become real bottleneck/blocker - this will fix them for good https://github.com/microsoft/fluentui-contrib/pull/155

**Review by marcosmoura — APPROVED** [review]

File: packages/react-tree-grid/src/components/TreeGrid/TreeGrid.component-browser-spec.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { test, expect } from '@playwright/experimental-ct-react';
-import { TreeGridExample } from './TreeGridExample.spec';
+import { TreeGridExample } from './TreeGridExample.component-browser-spec';
```
[the remark this answers is not in the document]
Comment (Hotell) [reply packages/react-tree-grid/src/components/TreeGrid/TreeGrid.component-browser-spec.tsx:3]: ha ! TIL https://github.com/microsoft/playwright/issues/18057,  this isn't the best DX but I understand the reasoning behind this. we should come up with some patterns to streamline this

File: packages/react-draggable-dialog/src/components/DraggableDialog/DraggableDialog.component-browser-spec.tsx [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
 
-const Example = () => {
-  return <div>Example</div>;
-};
-
 test.use({ viewport: { width: 500, height: 500 } });
 
 test('should render basic component', async ({ mount }) => {
-  const component = await mount(<Example />);
+  const component = await mount(<div>Example</div>);
```
Comment (Hotell) [inline packages/react-draggable-dialog/src/components/DraggableDialog/DraggableDialog.component-browser-spec.tsx:7]: looks like that previous pattern used wasn't capturing this test, thus we didn't run into any errors. now we do

https://github.com/microsoft/playwright/issues/18057

File: packages/react-draggable-dialog/playwright.config.ts [hunk 3]
```diff
@@ -47,9 +47,5 @@ export default defineConfig({
       name: 'firefox',
       use: { ...devices['Desktop Firefox'] },
     },
-    {
```
Comment (Hotell) [inline packages/react-draggable-dialog/playwright.config.ts:50]: https://github.com/microsoft/fluentui-contrib/issues/157

File: packages/react-tree-grid/src/components/TreeGrid/TreeGrid.component-browser-spec.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (bsunderhus) [reply packages/react-tree-grid/src/components/TreeGrid/TreeGrid.component-browser-spec.tsx:3]: Yeah, I wish I didn't had to create a separate file component 😅

**Review by bsunderhus — APPROVED** [review]

**Pull request merged**